modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/D3OrganizationChart.js

```diff
@@ -84,44 +79,49 @@ class D3OrganizationChart {
 		}
 	}
 
+	_handleKeyUp() {
+		this._multiSelectHandler.resetSelectableItems();
+	}
+
 	_addListeners() {
 		document.addEventListener('keydown', this._handleKeyDown);
-		document.addEventListener(
-			'keyup',
-			this._multiSelectHandler.resetSelectableItems
-		);
+		document.addEventListener('keyup', this._handleKeyUp);
 	}
 
 	addNodes(children, type, parentData) {
-		const parentId = getEntityId(parentData);
+		const parentId = parentData.id;
+
 		const formattedChildren = children.map((child) =>
 			formatChild(child, type)
 		);
-		let lastNodeAdded = null;
+
+		let firstNodeAdded = null;
 
 		this._root.each((d) => {
-			if (getEntityId(d.data) === parentId) {
+			if (d.data.id === parentId) {
```

Comment:
Hey @FabioDiegoMastrorilli, We try to avoid abbreviations. Could you explain what `d` is used for? I suspect it is for something from D3 and I don't have too much experience on it. Is this a known name for D3 users?

Reply:
@diegonvs I think `d` should be fine since that is a common syntax in d3 code.

Reply:
Hi @diegonvs, we've already discussed it [here](https://github.com/liferay-frontend/liferay-portal/pull/1311#discussion_r682513399). It's a common name for d3 nodes :)